apps/services/form-system/src/app/modules/applications/models/application.mapper.ts

```
@@ -9,6 +9,9 @@ import { SectionDto } from '../../sections/models/dto/section.dto'
 import { Application } from './application.model'
 import { ApplicationDto } from './dto/application.dto'
 import { ValueDto } from './dto/value.dto'
+import { ApplicationStatus, SectionTypes } from '@island.is/form-system/shared'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'SectionTypes'. Consider to delete it or rename it.

'SectionTypes' is defined here:


(lint/suspicious/noRedeclare)